### DISCUSSION TITLE
Notes

### DISCUSSION THREAD

**Discussion by Rasmus-Riis** [teslamate-org/teslamate#2492] [Ideas]
Is it possible to add notes to e.g your drives and then filter by them? 

Then would be a great addition if this is not possible.

I'm thinking among other things notes regarding having those God awful hubcaps on, that is supposed to save battery. It would be good to filter drives to see if they really gives a better efficiently or if it is just all wishful thinking.

But that's just one usecase. Notes on the drives when the wife is driving comes to mind. Then you would quickly see if she is such an angel as she says she is.
How much does it help to drive with HVAC turned off? 
The possibilities are endless...

Sorry in advance if this is possible. I haven't been able to find it.

**Comment by PCjabber** [comment]
It would be great to have a (searchable) comments/notes field for Drive data! Sometimes the numbers don't tell the entire story (traffic conditions, FSD usage, mood, etc).
